lib/block_account.go

```diff
@@ -21,6 +21,8 @@ import (
 
 const BlockAccountPrefixAddress string = "ba-address-"
 const BlockAccountPrefixCreated string = "ba-created-"
+const BlockAccountCheckpointPrefix string = "bac-ac-"
+const BlockAccountCheckpointByAddressPrefix string = "bac-aa-"
```

Comment:
What is `aa` mean?

Reply:
It's not straightforward. I don't have another idea yet. 

* `bac` is `BlockAccountCheckpoint`
* `ac` is `AccountCheckpoint`
* `aa` is `Account..Address`.

Reply:
@outsideris As you know, the middle part is only for making distinct key, anything can be possible. Please suggest good middle name ㅎㅎ